### PR TITLE
Add a 45 min option to the duration in the talk proposal form

### PR DESCRIPTION
**Pull request by alice6373**
## Add a tick box to the proposal form indicating 1st-time speaker
- [ ] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [x] **Enhancement**

## Description
Support program team's new requirement. 
Add a 45 min option to the duration in the talk proposal form that give the speaker more option to choose.

## Steps to Test This Pull Request
1. Go to 'MY PYCON'
2. Click on 'proposals' tab
4. Click on '+New Talk Proposal'
3. Scroll down
4. See tick box of *duration*

## More Information
**Screenshots**
- Frontend

image

- Django Administration
image

### CODE REVIEW

File: src/locale/zh_Hant/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -861,8 +861,10 @@ msgid "labels"
 msgstr ""
 
 #: proposals/models.py:272
+#, fuzzy
+#| msgid "additional speaker"
 msgid "first time speaker"
-msgstr "第一次擔任講者"
+msgstr "額外講者"
```
Comment (mattwang44) [inline src/locale/zh_Hant/LC_MESSAGES/django.po:867]:
```suggestion
msgid "first time speaker"
msgstr "第一次擔任講者"
```

File: src/locale/zh_Hant/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -719,7 +719,7 @@ msgstr "因應新型冠狀病毒疫情，若到八月疫情未趨緩，是否同
 
 #: proposals/forms/mixins.py:134
 msgid "Is it your first time to be the speaker of PyCon TW?"
-msgstr "此次是您第一次擔任 PyCon TW 的講者嗎？"
+msgstr ""
```
Comment (mattwang44) [inline src/locale/zh_Hant/LC_MESSAGES/django.po:722]: revert this change

File: src/pycontw2016/settings/base.py [hunk 3]
```diff
@@ -318,6 +318,7 @@ def node_bin(name):
     ('NOPREF', _('No preference')),
     ('PREF15', _('Prefer 15min')),
     ('PREF30', _('Prefer 30min')),
+    ('PREF45', _('Prefer 45min')),
```
Comment (mattwang44) [inline src/pycontw2016/settings/base.py:321]: it's better to put a brand new `TALK_PROPOSAL_DURATION_CHOICES` in `settings/production/pycontw-2021.py`.

File: src/locale/zh_Hant/LC_MESSAGES/django.po (hunk 1 above)
Comment (alice6373) [reply src/locale/zh_Hant/LC_MESSAGES/django.po]: Revise in 045d418bdee5359283dd4245bedbd8a042197dc3

File: src/locale/zh_Hant/LC_MESSAGES/django.po (hunk 2 above)
Comment (alice6373) [reply src/locale/zh_Hant/LC_MESSAGES/django.po]: Thank you. I revise it in 045d418

File: src/locale/zh_Hant/LC_MESSAGES/django.po (hunk 1 above)
Comment (alice6373) [reply src/locale/zh_Hant/LC_MESSAGES/django.po]: Thank you. I revised it in 045d418

File: src/pycontw2016/settings/base.py (hunk 3 above)
Comment (alice6373) [reply src/pycontw2016/settings/base.py]: Thanks. I moved it to settings/production/pycontw-2021.py. I also copy the same part to testing/pycontw-2021.py and checked that the result is correct.

Change setting file cmd : 
python /app/src/manage.py runserver 0.0.0.0:8000 --setting=pycontw2016.settings.testing.pycontw2021

**Review by mattwang44 — APPROVED** [review]
LGTM

**Comment by mattwang44** [comment]
[Fix ALL the bugs found during the proposal system testing](https://trello.com/c/j0ezAODh/24-fix-all-the-bugs-found-during-the-proposal-system-testing)
